src/styles/blocks/header.scss

```diff
@@ -0,0 +1,85 @@
+.header {
+  background-image: url(../images/header-background.png);
+  @include backgroundImage;
+
+  @include onTablet {
+    background-image: url(../images/background-image-medium.png);
+  }
+
+  @include onDesktop {
+    background-image: url(../images/header-background-big.png);
+  }
+
+  &__content {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    @include topPadding(0);
+    @include botPadding(0);
+    height: 100vh;
+    box-sizing: border-box;
+  }
+
+  &__top {
+    @include topActions;
+  }
+
+  &__logo {
+    display: flex;
+    align-items: center;
+
+    &-image {
+      height: 16px;
+      width: 130px;
+
+      @include onTablet() {
+        height: 24px;
+        width: 190px;
+      }
+    }
```

Comment:
if `header__logo-image` is an element of header then move styles to `.header` using `&__logo-image`, it would be more understandable when you're using bem
if this image is a modificator than it should be `&--image`